xbmc/music/tags/TagLibVFSStream.cpp

```diff
@@ -37,6 +39,17 @@ TagLibVFSStream::TagLibVFSStream(const std::string& strFileName, bool readOnly)
   {
     if (!m_file.Open(strFileName))
       m_bIsOpen = false;
+    else
+    { // Check if this stream is a Shoutcast, and skip scanning tags if it is.
+      // Shoutcast has its own metadata format that TagLib does not read
+      CFileCache* fileCacheItem = dynamic_cast<CFileCache*>(m_file.GetImplementation());
```

Comment:
Deactivating cache I still get the first file in the sequence as a CFileCache.
@Paxxi suggested a looping variant with IFile, but the inheritance chain is broken so I can't get that to work either.